integration-tests/hibernate-reactive-postgresql/src/test/java/io/quarkus/it/hibernate/reactive/postgresql/HibernateReactiveTest.java

```diff
@@ -18,6 +19,15 @@
 @TestHTTPEndpoint(HibernateReactiveTestEndpoint.class)
 public class HibernateReactiveTest {
 
+    @Test
+    public void reactiveFindNativeQuery() {
+        RestAssured.given().when()
+                .auth().preemptive().basic("scott", "jb0ss")
+                .get("/reactiveFindNativeQuery")
+                .then()
+                .statusCode(anyOf(is(200), is(204)));
```

Comment:
This check that there are no errors, but I would also check that the returned list contains the expected results.
For what we know, it could be empty or a list of `null`.

Reply:
My point here is that because the test method fetches all data, it could be empty.

Reply:
How could it be empty?  The db is populated at the beginning of the test. Maybe I'm missing something about how the test is working